### PR TITLE
Unblock Process after killing

### PR DESCRIPTION
**Pull request by EugenGanshorn**

### CODE REVIEW

**Review by SilvesterD — APPROVED** [review]

**Review by sp4ceb4r — CHANGES_REQUESTED** [review]
Thanks for your commit! The only thing I'd like to see here is minor change refactoring the duplicated  code introduced here.

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 </p>
 
 # shell-command
+[![Build Status](https://travis-ci.org/sp4ceb4r/shell-command.svg?branch=v0.2.0)](https://travis-ci.org/sp4ceb4r/shell-command)
```
Comment (sp4ceb4r) [inline README.md]: Remove this. Build status is displayed (L3).

If you want to change the status url to show the current branch, feel free.

File: src/Shell/Process.php [hunk 2]
```diff
@@ -543,6 +543,14 @@ public function kill($signal = SIGTERM)
...
         }
 
+        foreach ($this->pipes as $index => $pipe) {
+            if ($index === static::STDIN) {
+                continue;
+            }
+
+            stream_set_blocking($pipe, static::NON_BLOCKING);
+        }
+
```
Comment (sp4ceb4r) [inline src/Shell/Process.php:547]: Refactor the duplicated code into a method.

```php
protected function setStreamBlocking($blocking = static::NON_BLOCKING, $pipes = null)
{
    $pipes = $pipes ?: $this->pipes;
    // ...
}
```

**Review by sp4ceb4r — CHANGES_REQUESTED** [review]

File: src/Shell/Process.php (hunk 2 above)
Comment (EugenGanshorn) [reply src/Shell/Process.php:547]: done

**Comment by sp4ceb4r** [comment]
Thanks for you contribution!

**Review by sp4ceb4r — APPROVED** [review]

**Pull request merged**